lib/pdk/telemetry/util.rb

```diff
@@ -0,0 +1,16 @@
+module PDK
+  module Telemetry
+    module Util
+      def self.compute_os
+        require 'concurrent/configuration'
+        require 'concurrent/future'
+
+        Concurrent::Future.execute(executor: :io) do
+          require 'facter'
+          os = Facter.value('os')
+          "#{os['name']} #{os.dig('release', 'major')}"
```

Comment:
Did we figure out if `facter` is available at runtime to PDK itself?

Reply:
The simplest thing to do is adding facter 2.5.x as a dependency of PDK as it doesn't introduce any new dependencies.